### PR TITLE
Fix: fix all typo, indentations ready to deploy on master

### PR DESCRIPTION
**Pull request by suraj-aggarwal**
Fix: fix all typo, indentations ready to deploy on master

### CODE REVIEW

File: extra/utils/helper.js [hunk 1]
```diff
@@ -4,4 +4,4 @@ const validateEmail = (str) => {
     return regex.test(str)
 }
 
-export {validateEmail}
\ No newline at end of file
+export { validateEmail }
```
Comment (madhavBansal-successiveTech) [inline extra/utils/helper.js]: enter required

File: extra/utils/validation.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+        reveiwerName: 'madhav.bansal@successive.tech'
     },
     {
-        traineeName : 'anjali.shah@succesive.tech',
-        reveiwerName : 'pooja.thapa@successive.tech'
+        traineeName: 'anjali.shah@succesive.tech',
+        reveiwerName: 'pooja.thapa@successive.tech'
     }
-] 
+]
```
Comment (madhavBansal-successiveTech) [inline extra/utils/validation.js]: semicolon required

Comment (madhavBansal-successiveTech) [inline extra/utils/validation.js:10]: fix all linting issues like:
comma,semicolon etc.

File: src/controllers/trainee/Controller.ts [hunk 3]
```diff
@@ -16,109 +16,91 @@ class TraineeController {
...
       const options: object = { skip, limit, sort };
-      const filter: object = await this.initSearch(search);
+      const filter: object = await initSearch(search);
       let result;
-      if (Object.keys(filter).length) {
-        result = await this.userRepo.getAllRecord(filter, options);
-      } else {
-        result = await this.userRepo.getAllRecord(query, options);
-      }
+      const querySchema = Object.keys(filter).length !== 0 ? filter : query;
```
Comment (madhavBansal-successiveTech) [inline src/controllers/trainee/Controller.ts]: Object.keys(filter).length !== 0 can be written as Object.keys(filter).length

File: src/libs/utils/helper.ts [hunk 4]
```diff
@@ -1,7 +1,21 @@
+import * as queryString from 'query-string';
+
+
 const validateEmail = (str: string) => {
-    console.log('validateEmail', str);
-    const regex = /^[A-Za-z0-9._%+-]+@successive.tech$/;
-    return regex.test(str);
+  console.log('validateEmail', str);
```
Comment (madhavBansal-successiveTech) [inline src/libs/utils/helper.ts]: remove console

File: extra/utils/helper.js (hunk 1 above)
Comment (suraj-aggarwal) [reply extra/utils/helper.js]: Done

File: extra/utils/validation.js (hunk 2 above)
Comment (suraj-aggarwal) [reply extra/utils/validation.js:10]: Done

File: src/controllers/trainee/Controller.ts (hunk 3 above)
Comment (suraj-aggarwal) [reply src/controllers/trainee/Controller.ts]: Done

File: src/libs/utils/helper.ts (hunk 4 above)
Comment (suraj-aggarwal) [reply src/libs/utils/helper.ts]: Done

**Pull request merged**